tools/github_stats.py

```diff
@@ -227,3 +227,16 @@ def report(issues, show_urls=False):
         print()
         print('Issues (%d):\n' % n_issues)
         report(issues, show_urls)
+    print()
+    print()
+    print("""Previous GitHub Stats
+---------------------
+
+
+.. toctree::
+    :maxdepth: 1
+    :glob:
+
+    github_stats_*
+
+"""
```

Comment:
Missing `)`.

Reply:
fixed and force-pushed.  On one hand, I am glad the UI now shows force pushes for traceability, on the other hand it makes it harder to hide embarrassing mistakes like this ;)